GraphQL.md

```diff
@@ -19,47 +19,75 @@ POST https://yourdomain.com/graphql
 }
 ```
 
-where `rootObjectTypeName` is the name of an [Elide rootable](http://elide.io/pages/guide/01-start.html) object. Any object then can take any of 3 optional arguments.
+where `rootObjectTypeName` is the name of an [Elide rootable](http://elide.io/pages/guide/01-start.html) object. Any object then can take any of the 4 optional arguments.
 
-  * `op` (abbreviated for `operation`) is one of `FETCH`, `REPLACE`, `ADD`, or `DELETE`. The default value is `FETCH` when unspecified.
-  * `id` is the identifier for an object. If this value is unspecified the system assumes to be working on a collection of these objects. Otherwise, it is working on the specified id.
-  * `data` is the data input object that is used as the argument to the specified `op` argument. This argument is unused in the case of `FETCH` and `DELETE`.
+  * `op` (abbreviated for `operation`) is one of `UPSERT`, `REPLACE`, `REMOVE` or `DELETE`. The default value is `FETCH` when unspecified.
 
-Finally, the `exposedFields` is a specified list of values for the object(s) that the caller expects to be returned.
+     * `UPSERT` - inserts new or edits existing members of a relationship. It is the primary mechanism by which any updates will be made to the model.
+     * `REPLACE` - can be thought as a combination of `UPSERT` and `REMOVE`, wherein, any objects not specified in the data argument (that are visible to the user) will be removed.
+     * `REMOVE` - only disassociates a relationship. If the underlying store automatically deletes orphans, it will still be deleted.
+     * `DELETE` - both disassociates the relationship and it deletes the entity from the persistence store.
+     * `FETCH` - fetches an already existing entity from the persistent datastore. 
+  
+  * `ids` are used as the identifier for object(s). This field holds a list of ids. If this value is unspecified the system assumes to be working on a collection of these objects. Otherwise, it is working on the specified ids.
+  *  `data` is the data input object that is used as the argument to the specified `op` argument. This argument is unsupported in the case of `REMOVE`, `DELETE` or while _fetching_ an existing object.
+  *  `ids` is the id input object list that is used as the argument to the specified `op` argument. This argument is unsupported in the case of `UPSERT` and `REPLACE`.
+
+Finally, the `attributeName` is a specified list of values for the object(s) that the caller expects to be returned. 
 
 # Examples
-Below are several examples. For each, assume a simple model of `Book` and `Author`. Particularly, the schema for each can be considered:
+Below are several examples. For each, assume a simple model of `Book`, `Author` and `Publisher`. Particularly, the schema for each can be considered:
 
 ```java
-@Include(rootLevel = true)
 @Entity
+@Table(name = "book")
+@Include(rootLevel = true)
 public class Book {
     @Id public long id;
     public String title;
+    @OneToMany
```

Comment:
I think this is actually ManyToMany.

Multiple books can share the same author and a given author can have written multiple books.